data/stoa0255/stoa004/__cts__.xml

```diff
@@ -1,7 +1,8 @@
 <cts:work xmlns:cts="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:stoa0255.stoa004" groupUrn="urn:cts:latinLit:stoa0255" xml:lang="lat">
-  <cts:title xml:lang="lat">de Brevitate Vitae</cts:title>
+  <cts:title xml:lang="lat">De Brevitate Vitae</cts:title>
   <cts:edition urn="urn:cts:latinLit:stoa0255.stoa004.perseus-lat2" workUrn="urn:cts:latinLit:stoa0255.stoa004">
-    <cts:label xml:lang="eng">de Brevitate Vitae, Moral essays Vol 2</cts:label>
-    <cts:description xml:lang="eng">Seneca, Lucius Annaeus, ca. 4 B.C.-65 A.D, creator; Basore, John W. (John William), b. 1870, editor; Basore, John W. (John William), b. 1870, editor, translator</cts:description>
+    <cts:label xml:lang="lat">De Brevitate Vitae</cts:label>
+    <cts:description xml:lang="eng">Seneca, Lucius Annaeus. Moral Essays, Volume 2. Basore, John W. (John William), editor. 
```

Comment:
I think we are expanding names, right?

Reply:
hi @lcerrato yes we are, I will make those changes for the other files as well.